packages/bsky/src/api/app/bsky/graph/getList.ts

```diff
@@ -68,7 +70,29 @@ const hydration = async (
       params.hydrateCtx,
     ),
   ])
-  return mergeStates(listState, profileState)
+  const bidirectionalBlocks = await maybeGetBlocksForReferenceList({
+    ctx,
+    params,
+    skeleton,
+    listState,
+  })
+  return [listState, profileState, { bidirectionalBlocks }].reduce(
+    mergeStates,
+    {} as HydrationState,
+  )
+}
+
+const noBlocks = (input: RulesFnInput<Context, Params, SkeletonState>) => {
+  const { skeleton, hydration } = input
+  if (!hydration.bidirectionalBlocks) {
+    return skeleton
+  }
```

Comment:
This branch is fine, but IMO also cool to nix it.  If we want to keep it, can get rid of the defensive check below `hydration.bidirectionalBlocks?.get(creator)`.